crates/astria-sequencer/src/service/consensus.rs

```diff
@@ -131,28 +131,32 @@ impl Consensus {
             bail!("database already initialized");
         }
 
-        let genesis_state: GenesisState = serde_json::from_slice(&init_chain.app_state_bytes)
-            .expect("can parse app_state in genesis file");
+        let genesis_state: GenesisState = serde_json::from_str(
```

Comment:
i need to update to use from_bytes

Reply:
IMO not obfuscating that this is json is a good thing.